index.js

```diff
@@ -20,39 +20,27 @@
 //. purpose: the [`typeof`][1] operator and [`Object.prototype.toString`][2].
 //. Each has pros and cons, but neither supports user-defined types.
 //.
-//. This package specifies an [algorithm][3] for deriving a _type identifier_
-//. from any JavaScript value, and exports an implementation of the algorithm.
-//. Authors of algebraic data types may follow this specification in order to
-//. make their data types compatible with the algorithm.
+//. This package exports a function which derives a _type identifier_ from any
+//. JavaScript value, and a [specification][3] for customizing the type
+//. isdentifier of a value.
 //.
-//. ### Algorithm
```

Comment:
I removed the specification of "algorithm" and made it a specification on conforming to the algorithm. It was getting quite complex and I'm not sure if it has added value compared to just specifying the string format and where it should live. I'm curious about what you think, though.